buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -731,7 +731,7 @@ class BeamModulePlugin implements Plugin<Project> {
       // configurations because they are never required to be shaded or become a
       // dependency of the output.
       def compileOnlyAnnotationDeps = [
-        "com.google.auto.value:auto-value-annotations:1.6.3",
+        "com.google.auto.value:auto-value-annotations:1.7",
```

Comment:
When we do merge, I think it is helpful to have the dependency upgrade in an isolated commit. I notice you have some spotless commits to squash, but I would keep the dep upgrade separate.

Reply:
ack